pulumi/grapl/__main__.py

```diff
@@ -339,15 +339,15 @@ def main() -> None:
         pulumi_config.get(key="lightstep-access-token") or ""
     )
     lightstep_endpoint = pulumi_config.require(key="lightstep-endpoint")
-    lightstep_is_endpoint_secure = (
-        pulumi_config.get(key="lightstep-is-endpoint-secure") or "true"
+    lightstep_is_endpoint_insecure = (
+        pulumi_config.get(key="lightstep-is-endpoint-insecure") or "false"
```

Comment:
This could be simplified to `pulumi_config.get("lightstep-is-endpoint-secure", "false")`, in line with the normal Python behavior. The `key` keyword is kind of noisy here, given existing Python idioms around this kind of function.

Do these have to be strings? Because `pulumi.Config` has a `get_bool` function that returns booleans. It's probably better to have these as Booleans, and then convert them to strings at the point of ultimate consumption, if the consumer _has_ to have them as strings.

Reply:
Testing whether I can use booleans now.